src/main/webapp/view/transcript/transcript.js

```diff
@@ -171,3 +173,43 @@ function scrollToTopOfTranscript(transcriptLine) {
   const ulElementOffset = transcriptLine.parentElement.offsetTop;
   transcriptContainer.scrollTop = transcriptLine.offsetTop - ulElementOffset;
 }
+
+/**
+ * Returns the next transcript line based on `currentTimeMs`.
+ */
+function getNextTranscriptLine(currentTimeMs) {
+  const nextTranscript = currentTranscriptLine.nextElementSibling;
+  // Video is playing normally.
+  if (isWithinTimeRange(currentTimeMs, nextTranscript)) {
+    return nextTranscript;
+  }
+  return findClosestTranscriptLine(currentTimeMs);
```

Comment:
can you add a comment as to why you first search the next transcript line and then you search closest transcript line?

Reply:
I added 2 comments, thanks!